app/workers/search/index_to_elasticsearch_worker.rb

```diff
@@ -0,0 +1,12 @@
+module Search
+  class IndexToElasticsearchWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :high_priority
+
+    def perform(object_class, id)
+      object = object_class.safe_constantize.find(id)
```

Comment:
Minor: since you use `safe_constantize` you should probably account for the possibility of `object` being `nil`.

Reply:
Per our Slack convo, I changed it to `.constantize`.

To summarize it for future reference, `nil` should never be passed in since this worker is called from a model concern. It should only be called from an existing model that has the `Searchable` concern included, therefore `self.class.name` (`object_class`) should always be there. And if it does end up happening with `nil` (someone trying to invoke it without including it and passing a `nil` argument) it's going to blow up with `NoMethodError` which will prompt us to investigate how that happened and iterate from there.

And by using `.constantize` vs `.safe_constantize`, we get better error messaging if a non-existent (i.e. recently deleted) model name gets passed in - `NameError: wrong constant name FakeName` so we can debug that situation more easily.